### PR TITLE
Improvement on openapi2apigee

### PR DESCRIPTION
**Pull request by mat-613**
Hi,
Following the advice of @ssvaidyanathan i have made some fix to my initial PR for some improvement to the tool.

- Initialized xmlString in generatePolicies.js 
- Added removeHeaderAuthorization.js policy after the verifyAccessToken.js policy
- Added test and sample case
- Fix indent

### CODE REVIEW

File: lib/policy_templates/security/removeHeaderAuthorization.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+var builder = require('xmlbuilder')
+var random = require('../../util/random.js')
+
+module.exports = {
+  removeHeaderAuthorizationTemplate: removeHeaderAuthorizationTemplate,
+  removeHeaderAuthorizationGenTemplate: removeHeaderAuthorizationGenTemplate
+}
+
+function removeHeaderAuthorizationTemplate (options) {
+  var aysnc = options.async || 'false'
```
Comment (ssvaidyanathan) [inline lib/policy_templates/security/removeHeaderAuthorization.js:26]: 'aysnc' is assigned a value but never used

File: lib/policy_templates/security/removeHeaderAuthorization.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  var continueOnError = options.continueOnError || 'false'
```
Comment (ssvaidyanathan) [inline lib/policy_templates/security/removeHeaderAuthorization.js:27]: 'continueOnError' is assigned a value but never used.

File: lib/policy_templates/security/removeHeaderAuthorization.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  var enabled = options.enabled || 'true'
```
Comment (ssvaidyanathan) [inline lib/policy_templates/security/removeHeaderAuthorization.js:28]: 'enabled' is assigned a value but never used.

Comment (ssvaidyanathan) [reply lib/policy_templates/security/removeHeaderAuthorization.js:28]: Pls run `npm run test` to find the errors. The test doesnt run due to the above mentioned

Comment (mat-613) [reply lib/policy_templates/security/removeHeaderAuthorization.js]: Sorry @ssvaidyanathan, i have pushed the fixes and also the name on the new test was wrong!

File: lib/policy_templates/security/removeHeaderAuthorization.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+var builder = require('xmlbuilder')
+var random = require('../../util/random.js')
+
+module.exports = {
+  removeHeaderAuthorizationTemplate: removeHeaderAuthorizationTemplate,
+  removeHeaderAuthorizationGenTemplate: removeHeaderAuthorizationGenTemplate
+}
+
+function removeHeaderAuthorizationTemplate (options) {
+  var name = options.name || 'removeHeaderAuthorization-' + random.randomText()
```
Comment (ssvaidyanathan) [inline lib/policy_templates/security/removeHeaderAuthorization.js:26]: To be consistent with the other policies, lets add the async, continueonerror and enabled attributes

```
  var aysnc = options.async || 'false'
  var continueOnError = options.continueOnError || 'false'
  var enabled = options.enabled || 'true'
```

**Comment by ssvaidyanathan** [comment]
Released in v1.1.0

**Pull request merged**